script.js

```diff
@@ -0,0 +1,329 @@
+// Therefore hypotenus is fixed to a variable for calculations
+let hypotenus = length**2 + base**2;
+return Math.sqrt(hypotenus);
+};
+
+// print rightangled triangle
+// Print function
+console.log("hypotenus of a right angled triangle is " + rightangled(30, 15));
+
+
+
+
+// problem 9
+
+// Formula for body mass index is  BMI = weight (kg) / [height (m)]²
+
+// In a function set value of weight and meters 
+const bmi = function (weight, height){
+
+//  divide weight by height²
+let index = weight/height**2;
+return index;
+};
+
+console.log(bmi(85, 2));
+const index = bmi(85, 2);
+
+
+// set conditions in which bmi would be printed. Use conditionals 
+// if BMI is to a particalar range of numbers print accordingly
+// Print BMI
+// Underweight: <= 18.5
+
+ if (index <= 18-5){
+     console.log("underweight");
+ } 
+// // else if otherwise
+// //  * Healthy weight: <= 24.9
+ else if(index <= 24.9){
+   console.log("healthy" );
+}
+// // else if 
+// //  * Overweight: <= 29.9
+
+else if (index <= 29.9){
+    console.log("overweight");
+ } 
+// // Else 
+// //  * Obese: >= 30.0
+
+else if (index >= 30.0){
+  console.log("obese");
+ }
+ else {
+    console.log("no weight assigned");
+ };
+
+
+
+
+// problem 10
+//give function values 
+// function digitsSum(number){
+//     //convert to a string to make it be consoled
+//     const digitToString = number.toString();
+//     console.log(digitToString);
+   
+//     let 
+
+// }
+// console.log(digitsSum(1234));
+
+
+
+// problem 11
+// nth fibonacci number to be 6th in a sequence of 0, 1, 1, 2, 3, 5, 8, 13...
+// formula is Fn = Fn-1 + Fn-2
+
+// this is a list of fibbonacci sequence as an array
+const numlist = [0,1,1,2,3,5,8,13];
+// it will list out the numberin the 6th index when set to let  i= 6;
+  for (let i = 6; i < numlist.length; i++){
+    // console.log(); 
+ // a condition to print only the 6th number
+    if(numlist[i]===8){
+        // print nth number 
+        console.log("the 6th fibonacci number is " + numlist[i]);
+        // stop loop from running
+        break;
+    }
+  };
+
+
+
+
+
+// problem 12
+// the formula for calculating the roots  
+// x = (-b ± √(b² - 4ac)) / 2a
+
+// give values to the coefficients a, b, c
+
+
+// function equation(a, b, c, d){
+//     let d = (-b)**2,
+//  let answerX = (-b + d /2*a );
+
+//  let f =Math.sqrt(b**2 - 4*a*c);
+//  return answerX, d ;
+// }
+
+//print function
+
+//  console.log(equation(15, 10, 8, 3));
+
+
+
+// problem 13
+// first n natural numbers are 1, 2, 3, 4, 5, 6, 7, 8, 9...
+// a function where we give parameters to 
+
+function naturalNumbers(one, two, three, four, five, six, seven, eight, nine){
+    // calculate sum of squres
+    let sumOfSquares =  one**2 + two**2 + three**2 + four**2 + five**2 + six**2 + eight**2 + nine**2;
+    return sumOfSquares;
+
+};
+//print the answer
+console.log("sum of the squares of the first n natural numbers is " + naturalNumbers(1,2,3,4,5,6,7,8,9));

```

Comment:
Review this.

Your function should take a a single number and return sum of the square from 1 to that number.